osafw-app/App_Code/controllers/DevDBUpdates.cs

```diff
@@ -0,0 +1,149 @@
+// Demo Dynamic Admin controller
+//
+// Part of ASP.NET osa framework  www.osalabs.com/osafw/asp.net
+// (c) 2009-2023 Oleg Savchuk www.osalabs.com
+
+using System;
+using System.Collections;
+using System.Data.SqlTypes;
+using System.Drawing.Imaging;
+using System.Runtime.ConstrainedExecution;
+using static osafw.FwSelfTest;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace osafw;
+
+public class DevDBUpdatesController : FwDynamicController
```

Comment:
I see you are using FwDynamicController, but then using static list_table html?

Also, when using FwDynamicController or FwAdminController you need to override all unused Actions with something Throw New ApplicationException or redirect